bot/mc.go

```diff
@@ -4,43 +4,137 @@ import (
 	"fmt"
 	"log"
 	"os"
+	"strings"
+	"time"
 	"venova/sshcmd"
 
 	"github.com/gorcon/rcon"
 )
 
-func restartMinecraft() {
+type minecraftAction struct {
+	Message string
+	Command string
+}
+
+var minecraftActions = map[string]minecraftAction{
+	"up":      {Message: "The Minecraft Server has been brought up", Command: "up -d"},
+	"down":    {Message: "The Minecraft server has been brought down", Command: "down"},
+	"restart": {Message: "The Minecraft server has been restarted.", Command: "restart"},
+}
+
+func manageMinecraftCmd(ctx CommandCtx) error {
+	msg := ctx.Message.Message
+	sess := ctx.Session
+	args := ctx.Args
+	action, exists := minecraftActions[args[0]]
+	if !exists {
+		if err := ctx.Reply("Action is not available."); err != nil {
+			return err
+		}
+	}
+	if memberHasRole(msg.Member, mcRoleId) || msg.Author.ID == morthisId {
+		mcMsg, _ := sess.ChannelMessageSend(msg.ChannelID, "Attempting to modify the minecraft server...")
+		go func() {
```

Comment:
We probably don't need this in a goroutine anymore, do we? The command handler spawns this function in a goroutine so we can just do things inline.

Reply:
Also this will make error handling easier here, we can just return them.